plot_matplotlib.py

```diff
@@ -0,0 +1,76 @@
+import matplotlib.pyplot as pyplot
+
+from matplotlib.backends.backend_pdf import PdfPages
+
+
+def continuous_monitor_col(key, continuous_monitor):
+  """
+  For a given key, returns a list of data from continuous monitor entries
+  corresponding to that key.
+  """
+  return [data[key] for data in continuous_monitor]
+
+
+def plot(cm_data, file_prefix, open_graphs):
+  disk_utilization_params = ['xvdf utilization',
+                             'xvdb utilization',
+                             'xvdf write throughput',
+                             'xvdf read throughput',
+                             'xvdb write throughput',
+                             'xvdb read throughput']
+  memory_params = ['free heap memory',
+                   'free off heap memory']
+  monotasks_params = ['local running macrotasks',
+                      'macrotasks in network',
+                      'macrotasks in compute',
+                      'macrotasks in disk',
+                      'running monotasks',
+                      'gc fraction',
+                      'outstanding network bytes']
+  utilization_params = ['cpu utilization',
+                        'xvdf utilization',
+                        'xvdb utilization',
+                        'cpu system',
+                        'gc fraction']
+  xvdf_params = ['xvdf running disk monotasks',
+                 'xvdf write throughput',
+                 'xvdf read throughput',
+                 'xvdf utilization']
+  xvdb_params = ['xvdb running disk monotasks',
+                 'xvdb write throughput',
+                 'xvdb read throughput',
+                 'xvdb utilization']
+
+  def plot_params(params_to_plot, title):
+    """
+    Creates a matplotlib graph using continuous monitor data.
+    Time is the x axis and data corresponding to each parameter is used to
+    generate a new line on the line graph.
+    """
+    handles = []
+    time = continuous_monitor_col('time', cm_data)
+    for key in params_to_plot:
+      handle, = pyplot.plot(time, continuous_monitor_col(key, cm_data),
```

Comment:
It's the same as pyplot.plot()[0]. This was more a case of copy-pasta. I'll change it to [0]